frontend/src/components/RepositoryPanel.vue

```diff
@@ -220,13 +220,13 @@ export default defineComponent({
           state.repositoryConfig.sheetPathTemplate;
       }
 
-      // Update project schemaMigrationType field firstly.
+      // Update project schemaChangeType field firstly.
       if (
         isDev() &&
```

Comment:
If we want to catch up with this release then the `isDev` flag should be removed.

Reply:
Yep, it will be addressed in a subsequent PR.